1.data-exploration/scripts/1.model_data_visualization.py

```diff
@@ -86,7 +86,7 @@
 
 # ## Describe input data
 
-# In[9]:
+# In[7]:
```

Comment:
It looks like 8 is skipped in the code cell order, so recommend confirming this file is run sequentially in the notebook prior to nbconverting.